### PR TITLE
Prototype derive array element coordinates

### PR DESCRIPTION
**Pull request by GernotMaier**
This PR provides a prototype of the derive array element coordinates workflow.

The workflow steps are:
- assert input data using [schemas/array_coordinates_UTM.schema.yml](schemas/array_coordinates_UTM.schema.yml)
- convert coordinates from UTM to corsika coordinates (and removes all non-telescope elements from file)
- assert derive parameter values [schemas/array_coordinates.schema.yml](schemas/array_coordinates.schema.yml)

To run the example, do:
```
cwltool DeriveArrayElementCoordinates.cwl ../tests/resources/test_derive_array_elements_coordinates.yml
```
This will download the required input data and schemas and run the three steps.

See [here](https://view.commonwl.org/workflows/github.com/gammasim/workflows/blob/prototype-DeriveArrayElementCoordinates/workflows/DeriveArrayElementCoordinates.cwl) for a workflow graph.

Note:
- [schemas/MST_mirror_2f_measurements.schema.yml](schemas/MST_mirror_2f_measurements.schema.yml) is not need for above functionality; somehow landed in this PR.
- [tests/resources/test_derive_array_elements_coordinates.yml](tests/resources/test_derive_array_elements_coordinates.yml) contains links to this branch. This needs to be changed before merging.
- ready for review are only the following tools (ignore all others):
   - [workflows/tools/assert_data.cwl](workflows/tools/assert_data.cwl)
   - [workflows/tools/derive_array_elements_coordinates.cwl]
- this uses a locally produced docker container and not the latest from the github package repository (as the code from [this PR](https://github.com/gammasim/simtools/pull/490) is needed
- removed most workflow files (e.g., Receive, Validate, etc), as I think it is easier to add this as steps to the Derivation workflow.
- need to learn and understand if and how to run unittests. At this point the CI is restricted to cwl validation and yml linting.

### CODE REVIEW

File: schemas/MST_mirror_2f_measurements.schema.yml
Comment (orelgueta) [inline schemas/MST_mirror_2f_measurements.schema.yml:1]: So you decided to add this file twice? Once here and once is simtools? That seems less than ideal.

**Review by orelgueta — APPROVED** [review]
Just one comment about a duplicated file so I approve. I did not go in too much detail through the schema definition cause I have a feeling I went through it already on GitLab. Either way, from skimming it, it looks fine. I wonder a bit that we need to list the allowed units, but I can imagine why it is necessary/important.

File: schemas/MST_mirror_2f_measurements.schema.yml
Comment (GernotMaier) [reply schemas/MST_mirror_2f_measurements.schema.yml:1]: This is one of the reasons for the [discussions on repository organisation](https://github.com/gammasim/simtools/issues/516). See also my last comment there which might make this file obsolete in the simtools testing directory.

**Comment by GernotMaier** [comment]
> Just one comment about a duplicated file so I approve. I did not go in too much detail through the schema definition cause I have a feeling I went through it already on GitLab. Either way, from skimming it, it looks fine. I wonder a bit that we need to list the allowed units, but I can imagine why it is necessary/important.

The json schema requires to explicitly list the units. I've noticed that myself that this is not ideal.

**Pull request merged**